draft-ietf-mls-protocol.md

```diff
@@ -1898,10 +1898,11 @@ The validity of a LeafNode needs to be verified at a few stages:
   to add the client to the group
 * When a LeafNode is received by a group member in an Add, Update, or Commit
   message
-* When a client joining a group receives LeafNode objects for the other members
-  of the group in the group's ratchet tree
+* When a client joining a group receives LeafNode objects for the leaves of
+  of the group's ratchet tree
 
-The client verifies the validity of a LeafNode using the following steps:
+To verify that a LeafNode is valid regardless of its use, the client starts
+with the following steps:
```

Comment:
I'm not clear why this says "regardless of its use".  Maybe you mean "In all these cases...", but I would prefer leaving it unadorned to not imply that there's a restriction to those cases (in case another one comes up).

If you're going to say "starts with..." you need to have more steps that follow.

Reply:
Thanks for pointing this out! (We used to have a earlier version with more cases and more steps.)